alb.tf

```diff
@@ -32,7 +32,7 @@ resource "aws_route53_record" "consul" {
 
 # Create a new target group
 resource "aws_alb_target_group" "consul_ui" {
-  name     = "${replace(format("%.32s", replace("tf-c_ui-${data.aws_vpc.vpc.tags["Name"]}", "_", "-")), "/\\s/", "-")}"
+  #name_prefix = "${replace(format("%.32s", replace("tf-c_ui-${data.aws_vpc.vpc.tags["Name"]}", "_", "-")), "/\\s/", "-")}"
```

Comment:
I noticed that in addition to commenting this out it's changed from `name` to `name_prefix` -- does this need to have a `name_prefix` like the alb above?

Reply:
Good catch!